src/fides/api/api/deps.py

```diff
@@ -84,3 +86,11 @@ def get_privacy_request_service(
     messaging_service: MessagingService = Depends(get_messaging_service),
 ) -> PrivacyRequestService:
     return PrivacyRequestService(db, config_proxy, messaging_service)
+
+
+def get_dataset_service(db: Session = Depends(get_db)) -> DatasetService:
+    return DatasetService(db)
+
+
+def get_dataset_config_service(db: Session = Depends(get_db)) -> DatasetConfigService:
+    return DatasetConfigService(db)
```

Comment:
Adding the service dependencies to inject into the dataset config and dataset endpoints